runtime/jcl/module.xml

```diff
@@ -142,8 +142,21 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<group name="se20">
 				<include-if condition="spec.java20"/>
 			</group>
-			<group name="jfr">
+			<group name="jfr_jdk11">
+				<include-if condition="spec.flags.opt_jfr"/>
+				<exclude-if condition="spec.java12"/>
```

Comment:
This group also needs `<include-if condition="spec.java11"/>` to be consistent with cmake builds.
Similar comment applies to `jfr_jdk17` and `jfr_jdk21`.